### ISSUE TITLE
[bitnami/redis] Can redis sentinel work without PVCs?

### ISSUE THREAD

**Issue by wilsoniya** [bitnami/charts#6127]
Despite PVCs being given as a [requirement](https://https///github.com/bitnami/charts/blob/20a412fc343160b581d0cc55821dec7a2d502c3d/bitnami/redis/README.md#L36
) for using the redis chart, the chart templates seem to indicate that PVCs aren't actually required if `master` and replica persistence are disabled:
* https://https///github.com/bitnami/charts/blob/20a412fc343160b581d0cc55821dec7a2d502c3d/bitnami/redis/templates/redis-master-statefulset.yaml#L351
* https://https///github.com/bitnami/charts/blob/20a412fc343160b581d0cc55821dec7a2d502c3d/bitnami/redis/templates/redis-slave-statefulset.yaml#L350

So If I have a cluster without PVC support, will I be able to use the redis chart in sentinel mode?

**Comment by juan131** [comment]
Yes @wilsoniya, you can use the chart without persistence. That said, by doing that you're assuming that you could lose the data in your Redis installation if the pods are reallocated. 

By the way, I'm currently working on a major refactorization of the chart that will remove tons of technical debt (see https://github.com/bitnami/charts/pull/6102) and I hope it to be merged on Monday. I recommend you to wait for the new major version to if you plan to play with the Redis chart and the persistence.
